webrtc.html

```diff
@@ -2003,6 +2003,22 @@ <h4>Set the RTCSessionDescription</h4>
                                   to <var>rtcpTransport</var>.
                                 </p>
                               </li>
+                              <li>
+                                <p>Set the
+                                  <a>[[\IceRole]]</a> of <var>transport</var>
+                                  and <var>rtcpTransport</var> according to the
+                                  rules of [[RFC8445]].</p>
+                                <div class='note'>The rules of [[RFC8445]] that
+                                  apply here are:
+                                  <ul>
+                                    <li>If <a>[[\IceRole]]</a> is not <a data-link-for="RTCIceRole">unknown</a>, do not modify <a>[[\IceRole]]</a>.
+                                    <li>If <var>description</var> is a local offer,
+                                      set it to <code><a data-link-for="RTCIceRole">controlling</a></code>.</li>
+                                    <li>If <var>description</var> is a remote offer, and contains "a=ice-lite", set <a>[[\IceRole]]</a> to <code><a data-link-for="RTCIceRole">controlling</a></code>.</li>
+                                    <li>If <var>description</var> is a remote offer, and does not contain "a=ice-lite", set <a>[[\IceRole]]</a> to <code><a data-link-for="RTCIceRole">controlled</a></code>.</li>
+                                  </ul>
+                                  This ensures that <a>[[\IceRole]]</a> always has a value after the first offer is processed.
```

Comment:
Does this need to get rolled back? ;)

Reply:
Seems we're safe against rollback ... rollback will either destroy the transport (initial offer) or leave it in a previous state, which will have IceRole set. No text needed.